plugins/modules/cloud/misc/proxmox_snap.py

```diff
@@ -38,6 +38,17 @@
       - For removal from config file, even if removing disk snapshot fails.
     default: false
     type: bool
+  unbind:
+    description:
+      - This option only applies to LXC containers.
+      - Allows to snapshot a container even if it has configured mountpoints.
+      - Temporarily disables all configured mountpoints, takes snapshot, and finally restores original configuration.
+      - If running, the container will be stopped and restarted to apply config changes.
+      - Due to restrictions in the Proxmox API this option can only be used authenticating as C(root@pam) with I(api_password), API tokens do not work either.
+      - See U(https://pve.proxmox.com/pve-docs/api-viewer/#/nodes/{node}/lxc/{vmid}/config) (PUT tab) for more details.
+    default: false
+    type: bool
+    version_added: 5.6.0
```

Comment:
Unfortunately 5.6.0 is already out, so this needs to be updated to 5.7.0:
```suggestion
    version_added: 5.7.0
```